sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
@tamird to the rescue, I think: https://github.com/github/linguist/commit/d17222ec916ca8682fa77cae6a4351de004fa894

This seems to indicate that if the first line of a `.go` file includes `Code generated by`, it's considered auto-generated. Wanna give it a try?